### PR TITLE
Add URL for darts portal to test env

### PR DESCRIPTION
**Pull request by jackmaloney**

### CODE REVIEW

File: apps/darts-modernisation/darts-api/test.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ spec:
       ingressHost: darts-api.test.platform.hmcts.net
       environment:
         DARTS_GATEWAY_URL: https://darts-gateway.test.platform.hmcts.net
+        DARTS_PORTAL_URL: https://darts.test.apps.hmcts.net
```
Comment (davet1985) [inline apps/darts-modernisation/darts-api/test.yaml:13]: This is already set via the main chart, I double-checked, see below.

```
√ darts-api DMP-2005-event-logging % k -n darts-modernisation describe po darts-api-java-6f46b4fbc6-c5tt6 | grep DARTS_PORTAL_URL
      DARTS_PORTAL_URL:                   https://darts.test.apps.hmcts.net
```

**Review by scott-robertson1 — APPROVED** [review]

File: apps/darts-modernisation/darts-api/test.yaml (hunk 1 above)
Comment (jackmaloney) [reply apps/darts-modernisation/darts-api/test.yaml:13]: ok, so we can remove these URLs from all flux config?

Comment (davet1985) [reply apps/darts-modernisation/darts-api/test.yaml:13]: No, some of them need to be there due to the value of the `.Values.global.environment` variable that is used by the charts. For example, for staging the value is `stg` which would produce an incorrect value of `https://darts.stg.apps.hmcts.net` for `DARTS_PORTAL_URL`.

Comment (davet1985) [reply apps/darts-modernisation/darts-api/test.yaml:13]: Could be removed from `demo` but the others are required. Tbh it does no harm, just won't make any difference.

**Review by davet1985 — APPROVED** [review]

**Pull request merged**